modules/porous_flow/src/materials/PorousFlow1PhaseP.C

```diff
@@ -29,7 +31,10 @@ PorousFlow1PhaseP::PorousFlow1PhaseP(const InputParameters & parameters)
     _porepressure_varnum(coupled("porepressure")),
     _p_var_num(_dictator.isPorousFlowVariable(_porepressure_varnum)
                    ? _dictator.porousFlowVariableNum(_porepressure_varnum)
-                   : 0)
+                   : 0),
+    _pc_uo(parameters.isParamSetByUser("capillary_pressure")
+               ? &getUserObject<PorousFlowCapillaryPressure>("capillary_pressure")
+               : nullptr)
```

Comment:
This is a bit of a hack so that I didn't break the deprecated materials. This can be changed to just `_pc_uo(getUserObject<>())` when the deprecated materials are removed